sklearn/utils/tests/test_array_api.py

```diff
@@ -546,7 +536,7 @@ def test_get_namespace_and_device():
     namespace, is_array_api, device = get_namespace_and_device(some_torch_tensor)
     assert namespace is get_namespace(some_numpy_array)[0]
     assert not is_array_api
-    assert device is None
+    assert device.type == "cpu"
```

Comment:
I think the comment above needs to be updated because it mentions **no device** whereas now we have **cpu**